src/pages/dashboard/OperatorHbarChart/OperatorHbarChart.tsx

```diff
@@ -13,17 +13,19 @@ const colorsByCompannies: { [index: string]: string } = {
   מטרופולין: '#FF8500',
 }
 
-const excludeOperators = [/רכבת ישראל/, /^מוניות.*/, /^ירושלים-.*/, /^ירושלים -.*/]
+// const excludeOperators = [/רכבת ישראל/, /^מוניות.*/, /^ירושלים-.*/, /^ירושלים -.*/]
```

Comment:
delete this line please

Reply:
delete.
applied regex filtering of dashboard ByDay to ByHour
extracted reg to Dashboard component to apply 1 param on both childs.